### PR TITLE
add Gas-water-solvent test case

### PR DESCRIPTION
**Pull request by totto82**
Test case for

https://github.com/OPM/opm-simulators/pull/4568

### CODE REVIEW

File: spe1_solvent/SPE1CASE2_GASWATER_SOLVENT.DATA [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+ 
+---------------------------------------------------------------------------
+------------------------ SPE1 - CASE 1 ------------------------------------
+---------------------------------------------------------------------------
+
+RUNSPEC
+-- -------------------------------------------------------------------------
+
+TITLE
+   SPE1 - CASE 1
```
Comment (GitPaean) [inline spe1_solvent/SPE1CASE2_GASWATER_SOLVENT.DATA:28]: file name is SPE1CASE2, so it should be consistent.

Comment (GitPaean) [reply spe1_solvent/SPE1CASE2_GASWATER_SOLVENT.DATA:28]: and it looks like all the DATA files in the folder share the same issue.

**Comment by GitPaean** [comment]
a suggestion for the organizing of the cases, can the cases be based on an existing test case and only contain the minimal and necessary difference for the testing purpose, so that we can look at the result and tell the difference in the results that result from the difference. 

Basically to demonstrate the effect of the feature in a clear way. 

When the cases have multiple difference, it is difficult to see how certain features (like foam or solvent) affect the result.

**Comment by totto82** [comment]
Now the test case is the same as SPE1CASE2_GASWATER but with solvent injection.

**Comment by GitPaean** [comment]
Thanks for the update. I will have a look at the case and test a little bit with it.

**Comment by GitPaean** [comment]
It generally looks good. But can you reduce the editing between `SPE1CASE2_GASWATER_SOLVENT.DATA` and `SPE1CASE2_GASWATER.DATA`? Most likely, it can be because the new case is based on a different version. 

It is no big deal, but when there are text moving around, comparison will be a little harder, at least visually.

Other than that, the test case can go in.

**Comment by totto82** [comment]
Now it is rebased on the SPE1CASE2_GASWATER.

**Comment by GitPaean** [comment]
Thanks for the update. I will just run one jenkins test and then get it in.

**Comment by GitPaean** [comment]
jenkins build this please

**Comment by GitPaean** [comment]
Thanks for the update. I am getting this PR in now.

**Pull request merged**